bundles/org.openhab.binding.upb/src/main/java/org/openhab/binding/upb/Constants.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.upb;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.type.ChannelTypeUID;
+
+/**
+ * Common constants used in the binding.
+ *
+ * @author Marcus Better - Initial contribution
+ */
+@NonNullByDefault
+public final class Constants {
```

Comment:
This is probably a remnisant of the older version of the binding. Since then we changed to have all files that are used in other projects to be in the `internal` package. For bindings this means all files should be in the internal package. So yes please move these files to the internal package.